kafka-connect/kafka-connect/src/main/java/org/apache/iceberg/connect/data/Utilities.java

```diff
@@ -28,6 +28,8 @@
 import java.util.Set;
 import java.util.UUID;
 import java.util.stream.Collectors;
+import org.apache.hadoop.conf.Configuration;
```

Comment:
We want to avoid dependencies on Hadoop. Take a look at how we initialize the Hadoop config in `Utilities` using reflection.